.travis.yml

```diff
@@ -16,7 +16,6 @@ matrix:
       env: TESTSCRIPT=test:everything DB="oracledb mssql mysql mysql2 postgres sqlite3" KNEX_TEST_TIMEOUT=60000
 install:
   - npm i
-  - (echo $DB | grep oracledb) && npm install oracledb@3.1.2 || true
```

Comment:
wait, whole line shouldn't be removed :D. just the `@3.1.2` part, so that latest driver version is picked up.